lib/jsdom/virtual-console.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+
+var EventEmitter = require("events").EventEmitter;
+
+var vConsole = {};
+
+var emitters = {};
+var windows = [];
+
+// Returns an EventEmitter that emits on any console events
+vConsole.getVirtualConsole = function(window) {
+  var virtualConsole = new EventEmitter();
+
+  saveEmitterForWindow(virtualConsole, window);
+
+  virtualConsole.sendTo = function(anyConsole) {
+
+    if (!isValidConsole(anyConsole, window)) {
+      var msg = "sendTo was not passed a compliant console";
+      throw new Error(msg);
+    }
+
+    var vc = vConsole.getVirtualConsole(window);
```

Comment:
You can just use `this` instead, when this becomes a method on `VirtualConsole`.